src/react/components/pages/editorPage/canvas.tsx

```diff
@@ -351,38 +376,87 @@ export default class Canvas extends React.Component<ICanvasProps, ICanvasState>
      */
     private updateEditor = () => {
         this.deleteAllRegions();
+        // We need to check if we're looking for a video or image
+        if (this.props.selectedAsset.asset.type === AssetType.Image) {
+            this.loadImage();
+        } else if (this.props.selectedAsset.asset.type === AssetType.Video) {
+            this.loadVideo();
+        } else {
+            // We don't know what type of asset this is?
+            throw new Error(strings.editorPage.assetError);
```

Comment:
FYI - we have a special `AppError` that we are throwing in other areas that handle error localization in 1 place.  Do a search for AppError and create a new ErrorCode that can then participate in the localization.

Reply:
Added new Canvas error code